test/chpldoc/globals/variablepatterns.doc.chpl

```diff
@@ -8,3 +8,4 @@ var g = forall i in 1..10 do i;
 var h = [i in 1..10] i;
 var i = try! x;
 var j = try x;
+var k = foo("this is", a, "function", call);
```

Comment:
Rather than restoring the future you just resolved, I think it would be better to just add a separate test with just this case in it

Reply:
Sure :)

Reply:
Hi Lydia, I have added a new future, please review. Thanks :)